packages/app/components/Queue/Student/QuestionForm.tsx

```diff
@@ -74,7 +74,7 @@ export default function QuestionForm({
     question?.text || ""
   );
   const [questionGroupable, setQuestionGroupable] = useState<boolean>(
-    question?.groupable === undefined || question?.groupable
+    (question?.groupable !== undefined && question?.groupable) || false
```

Comment:
I might not understand typescript but is the `|| false` necessary here?

Reply:
^^

Reply:
fixed mb